README.md

```diff
@@ -84,11 +84,17 @@ See the `docker-compose.yml` example below.
 ```
 
 ## Receiving MLAT results
-If you run the `mlathub` container, or if you want to make the MLAT results available to your visualization software (piaware, tar1090, etc), you need to tell these containers to connect to the MLAT results port(s) of the `multifeeder` container. For example, using the settings in the sample `docker-compose.yml` (above) as a guideline:
+If you run the `mlathub` container, or if you want to make the MLAT results available to your visualization software (piaware, tar1090, etc), you need to tell these containers to connect to the MLAT results port(s) of the `multifeeder` container. For example, using the settings in the sample `docker-compose.yml` (above) as a guideline, append the following to your `mlathub` configuration:
 ```
       - READSB_NET_CONNECTOR=...;multifeeder,39000,beast_in;multifeeder,39001,beast_in;multifeeder,39002,beast_in
 ```
 
+To update tar1090 to use a specific feed from the `multifeeder` container, update the following environment variables:
+```
+      - MLATHOST=multifeeder
+      - MLATPORT=39000 #or use 39001 or 39002 based on your multifedder MLAT_CONFIG config
+```
```

Comment:
I don't know about introducing MLATHOST/MLATPORT. Those get included into READSB_NET_CONNECTOR like this:
```
    READSB_CMD+=("--net-connector=${MLATHOST},${MLATPORT},beast_in")
```

So they don't do anything different than line 89.

Reply:
The distinction I was trying to make is that in a situation where someone is not using `mlathub` they can specify a specific port from the `multifeeder`. This may be common if someone is moving off `adsbx` as the mlat client for tar1090, the current default in the gitbook. I ended up spinning up `mlathub` for my stack, but not sure that’ll be everyone’s inclination.